### DISCUSSION TITLE
Two solids in contact

### DISCUSSION THREAD

**Discussion by mischu80** [OpenRadioss/OpenRadioss#2716] [Users]
Hello, I started today my "Openradioss-Journey" (using gmsh and Paraview).
I am interested in contact mechanics or plastic deformation of solids in contact. I tried the tube impact example, which is modeled using shells. I am wondering if there exist an accessible  simple example for solids (!), e.g. two cylinder pressed to each other, modeled by tetrahedral meshes with time dependent load or constant velocity of approach . I just want to understand the general logic of the starter files for this class of problems.

Thank you for any hint

M

**Comment by rzadki22** [comment]
Hello mischu80,
you can find a lot of interesting examples and input files on the official altair website. Probably you are looking for something like:
https://2022.help.altair.com/2022.3/hwsolvers/rad/topics/solvers/rad/brake_example_r.htm
but please be aware that the gmsh is highly limited software for a radioss purpose and you can encounter bumpy troubles with this preprocessor. I wish you best luck!
Best regards,
Filip
